core/css.js

```diff
@@ -219,15 +219,13 @@ Blockly.Css.CONTENT = [
     'cursor: url("<<<PATH>>>/handopen.cur"), auto;',
     'cursor: grab;',
     'cursor: -webkit-grab;',
-    'cursor: -moz-grab;',
```

Comment:
Modern Mozilla browsers support grab and grabbing.  -moz prefix no longer required.